src/components/secret-display/SecretDisplayCard.js

```diff
@@ -0,0 +1,100 @@
+import { times } from 'lodash';
+import React, { useMemo } from 'react';
+import LinearGradient from 'react-native-linear-gradient';
+import ShadowStack from 'react-native-shadow-stack/dist/ShadowStack';
+import styled from 'styled-components/primitives';
+import { Centered, ColumnWithMargins, Row, RowWithMargins } from '../layout';
+import { Text } from '../text';
+import WalletTypes from '@rainbow-me/helpers/walletTypes';
+import { colors, fonts, padding, position } from '@rainbow-me/styles';
+
+const BackgroundGradient = styled(LinearGradient).attrs({
+  colors: colors.gradients.offWhite,
+  end: { x: 0.5, y: 1 },
+  start: { x: 0.5, y: 0 },
+})`
+  ${position.cover};
+`;
+
+const CardShadow = styled(ShadowStack).attrs({
+  ...position.coverAsObject,
+  borderRadius: 25,
+  shadows: [
+    [0, 10, 30, colors.dark, 0.1],
+    [0, 5, 15, colors.dark, 0.04],
+  ],
+})`
+  elevation: 15;
+`;
+
+const Content = styled(Centered)`
+  ${padding(19, 30, 24)};
+  border-radius: 25;
+  overflow: hidden;
+  z-index: 1;
+`;
+
+const GridItem = styled(Row).attrs({
+  align: 'center',
+})`
+  height: ${fonts.lineHeight.looser};
+`;
+
+const GridText = styled(Text).attrs(({ weight = 'semibold' }) => ({
+  lineHeight: 'looser',
+  size: 'lmedium',
+  weight,
+}))``;
+
+function SeedWordGrid({ seed }) {
```

Comment:
this component creates a slightly unintuitive layout of nested columns and rows, but its necessary to achieve the layout of the grid in the design spec... 

specifically its needed so that the numbers "align right" within their own column like this 
<img width="444" alt="Screen Shot 2020-09-02 at 6 20 31 PM" src="https://user-images.githubusercontent.com/1325144/92043167-91a31100-ed49-11ea-982e-9096fe512f3f.png">